[2022-02-09 08:59:41,774] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-12-01T23:00:00+00:00 [queued]>
[2022-02-09 08:59:41,794] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-12-01T23:00:00+00:00 [queued]>
[2022-02-09 08:59:41,794] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 08:59:41,794] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 08:59:41,794] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 08:59:41,814] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2020-12-01 23:00:00+00:00
[2022-02-09 08:59:41,823] {standard_task_runner.py:52} INFO - Started process 297 to run task
[2022-02-09 08:59:41,832] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2020-12-01T23:00:00+00:00', '--job-id', '477', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingestee_gcs_dag.py', '--cfg-path', '/tmp/tmppmjcc3a7', '--error-file', '/tmp/tmpcejrq1wt']
[2022-02-09 08:59:41,834] {standard_task_runner.py:77} INFO - Job 477: Subtask local_to_gcs_task
[2022-02-09 08:59:41,969] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-12-01T23:00:00+00:00 [running]> on host cc021fb126ed
[2022-02-09 08:59:42,051] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 08:59:42,094] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2020-12-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-12-01T23:00:00+00:00
[2022-02-09 08:59:50,382] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 08:59:50,395] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20201201T230000, start_date=20220209T085941, end_date=20220209T085950
[2022-02-09 08:59:50,430] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 08:59:50,535] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-09 09:15:22,806] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-12-01T23:00:00+00:00 [queued]>
[2022-02-09 09:15:22,847] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-12-01T23:00:00+00:00 [queued]>
[2022-02-09 09:15:22,847] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:15:22,847] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:15:22,848] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:15:22,899] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2020-12-01 23:00:00+00:00
[2022-02-09 09:15:22,916] {standard_task_runner.py:52} INFO - Started process 432 to run task
[2022-02-09 09:15:22,940] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2020-12-01T23:00:00+00:00', '--job-id', '576', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingesteee_gcs_dag.py', '--cfg-path', '/tmp/tmpq4t8uaau', '--error-file', '/tmp/tmps0u_il8i']
[2022-02-09 09:15:22,942] {standard_task_runner.py:77} INFO - Job 576: Subtask local_to_gcs_task
[2022-02-09 09:15:23,054] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-12-01T23:00:00+00:00 [running]> on host 75953ff4f3d5
[2022-02-09 09:15:23,167] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:15:23,213] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2020-12-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-12-01T23:00:00+00:00
[2022-02-09 09:15:30,735] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 09:15:30,749] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20201201T230000, start_date=20220209T091522, end_date=20220209T091530
[2022-02-09 09:15:30,786] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:15:30,826] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
